k8s/workloads/careers/careers-ingress-prod.yaml

```
@@ -0,0 +1,60 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ingress-nginx
+  namespace: careers-prod
+  labels:
+    app: careers
+spec:
+  releaseName: careers-ingress-nginx
+  chart:
+    repository: https://kubernetes.github.io/ingress-nginx
+    name: ingress-nginx
+    version: 3.23.0
+  values:
+    controller:
+      useIngressClassOnly: true
+      enableCustomResources: false
+      autoscaling:
+        enabled: true
+        minReplicas: 1
+        maxReplicas: 4
+        targetCPUUtilizationPercentage: 80
+        targetMemoryUtilizationPercentage: 80
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 200m
+          memory: 256Mi
+      admissionWebhooks:
+        enable: false
+      scope:
+        enabled: true
+      config:
+        enable-vts-status: true
+        force-ssl-redirect: true
+        # Rate limitting: defaults x7
+        limit-connections: "140"
+        limit-rpm: "1400"
+        limit-rps: "84"
+        proxy-body-size: 4m
+        proxy-buffer-size: 16k
+        # restrict this to the IP addresses of ELB
+        proxy-real-ip-cidr: "0.0.0.0/0"
```

I think we can set this to the subnet cidr

---

That makes sense, I copy/pasted this from another deployment, so we could make that change across all our deployments if that makes sense. Seems like this is the default, and the recommendation is the vpc ipv4 cidr -- https://kubernetes.github.io/ingress-nginx/user-guide/miscellaneous/

---

https://github.com/mozilla-it/itse-apps-prod-1-infra/blob/03f94a08515f6d4fc671bb1e95105ada2742b751/k8s/workloads/etherpad/etherpad-ingress.yaml#L42
```suggestion
        proxy-real-ip-cidr: "172.16.0.0/16"
```